onlineapi/build.gradle

```diff
@@ -26,6 +26,8 @@ dependencies {
     implementation 'com.squareup.retrofit2:converter-gson:2.3.0'
     implementation 'com.jakewharton.retrofit:retrofit2-kotlin-coroutines-adapter:0.9.2'
     implementation 'io.github.inflationx:viewpump:1.0.0'
+    implementation "org.jetbrains.anko:anko-commons:0.10.6"
```

Comment:
if this is only for logging purpose, then I would recommend leave it out. 

we should keep the onlinapi library as lean and mean as possible. we can do normal Log.d() calls.